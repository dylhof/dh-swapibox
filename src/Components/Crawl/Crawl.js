import React from 'react';

const Crawl = ({ crawl, title, date }) => {
  return (
    <div>
      <p className='crawl'>{crawl}</p>
      <p className='crawl'>{title}</p>
      <p className='crawl'>{date}</p> 
    </div>
  )
}

export default Crawl;