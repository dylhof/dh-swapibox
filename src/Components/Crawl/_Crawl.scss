@keyframes crawl {
	0% {
    top: -100px;
    transform: rotateX(20deg)  translateZ(0);
  }
	100% { 
    top: -6000px;
    transform: rotateX(25deg) translateZ(-2500px);
  }
}

.crawl-section {
  display: flex;
  flex-direction: column;
  align-items: center;
  position: relative;
  height: 1200px;
  perspective: 400px;
  
  .crawl-div{
    position: relative;
    top: 100px;
    animation: crawl 200s linear;
    width: 75%;
  }
  .crawl-title,
  .choose-category {
    @include text($swFontColor, 75px, center);
    margin: 10px;
    text-shadow: 6px 6px black;
  }
  .crawl {
    @include text($swFontColor, 50px, center);
    text-shadow: 6px 6px black;
  }


}