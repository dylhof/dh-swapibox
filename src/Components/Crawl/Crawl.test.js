import React from 'react';
import { shallow } from 'enzyme';
import Crawl from './Crawl';

const mockCrawl = 'It is a period of civil war.';

describe('Crawl', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(
      <Crawl crawl={mockCrawl} /> 
    )
    expect(wrapper).toMatchSnapshot();
  })
})