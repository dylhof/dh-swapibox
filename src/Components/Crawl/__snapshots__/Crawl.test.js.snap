// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Crawl should match the snapshot 1`] = `
<section
  className="crawl-section"
>
  <div
    className="crawl-div"
  >
    <p
      className="crawl-title"
    />
    <p
      className="crawl"
    />
    <p
      className="crawl"
    >
      It is a period of civil war.
    </p>
    <p
      className="choose-category"
    >
      Choose A Category...
    </p>
  </div>
</section>
`;
